src/rules/prefer-repository-shorthand.ts

```diff
@@ -0,0 +1,103 @@
+import type ESTree from "estree";
+import type { JSONLiteral } from "jsonc-eslint-parser/lib/parser/ast.js";
```

Comment:
[Request] No need to import from a sub-path, we can import a namespace the way `createRule.ts` does:

```suggestion
import type { AST as JsonAST } from "jsonc-eslint-parser/lib/parser/ast.js";
```

Reply:
Fixed